DQM/CSCMonitorModule/plugins/CSCDQM_StripClusterFinder.cc

```diff
@@ -124,7 +125,7 @@ namespace cscdqm {
 
     //              SEARCHING PARAMETERS OF THE CLASTERS (LEFT DOWN & RIGHT UP)
 
-    for (i = 0; i < MEStripClusters.size(); i++) {
+    for (uint32_t i = 0; i < MEStripClusters.size(); i++) {
       if (MEStripClusters[i].localMax.empty()) {
         std::cout << "!!!Warning Cluster has'nt local Maxima" << std::endl;
```

Comment:
All other `cout`'s are commented out in the code

Reply:
The commented out `cout` s all appear to be debugging statements where this one is issuing a warning. If anything, I'd say this should be converted to `edm::LogWarning` instead of being commented out.